ProcessMaker/Http/Controllers/Api/ProcessRequestController.php

```diff
@@ -839,4 +839,44 @@ public function endEventDestination(ProcessRequest $request)
 
         return response()->json(['message' => __('End event found'), 'data' => $data]);
     }
+
+    /**
+     * This endpoint returns requests by case number
+     *  
+     * @param Request $request
+     *
+     * @return ApiCollection
+     */
+    public function getRequestsByCase(Request $request)
+    {
+        $user = Auth::user();
+
+        $case_number = $request->input('case_number', 0);
+        $query = ProcessRequest::forUser($user);
+        $includes = $request->input('include', '');
+        foreach (array_filter(explode(',', $includes)) as $include) {
+            if (in_array($include, ProcessRequest::$allowedIncludes)) {
+                $query->with($include);
+            }
+        }
+
+        if ($advancedFilter = $request->input('advanced_filter', '')) {
```

Comment:
What is the purpose of this advanced filter?

Reply:
this is used, when is filter by status column